packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx

```diff
@@ -276,10 +277,14 @@ export const AddObjectMenu = (): JSX.Element => {
           uri: modelUri,
           modelType: modelType ?? ext.toUpperCase(),
         };
-
+        //does this model have an animation component if it does also add the animation component
```

Comment:
Where's the check for if it has animations? Looks like this add animations to all new gltf regardless.